front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -332,6 +332,18 @@
           white-space: nowrap;
           overflow: hidden;
           text-overflow: ellipsis;
+          display: flex;
+          align-items: center;
+          .train-projected {
+            height: 1.25rem;
+            width: 1.625rem;
+            position: relative;
+            svg {
+              position: absolute;
+              top: -0.125rem;
+              left: -0.125rem;
```

Comment:
in some case use negative values for positionning an absolute element could lead the element to expand his  size, tryc to set it to 0 and then apply a transformation

Reply:
Done